### ISSUE TITLE
Would be possible to support sysutils/mtpfs?

### ISSUE THREAD

**Issue by lebarondemerde** [vermaden/automount#10]
Hello!

Would be possible to have support to [sysutils/mtpfs](http://www.freshports.org/sysutils/mtpfs/) in automount?

Thanks!

**Comment by vermaden** [comment]
Theoretically yes, but ... The only device that shows in the `/dev` is `/dev/ugen0.3` which is link to `/dev/usb/0.3.0` device. The `simple-mtpfs` is not run against that device but 'generally'.

I can list the devices with `simple-mtpfs` like that:
```
% simple-mtpfs --list-devices
1: LG Electronics Inc.LG G Flex 2
```

But imagine attaching TWO same or very similar phones at the same time, with `simple-mtpfs --list-devices` You are not able to distinguish them and 'link' them to the device in `/dev` directory. So I can not attach an event to `devd(8)` to the device that appears in `/dev` directory.

So in short, no.

Regards,
vermaden

**Comment by lebarondemerde** [comment]
Ok, got it. 

Thanks!

**Comment by wulf7** [comment]
> But imagine attaching TWO same or very similar phones at the same time, with simple-mtpfs --list-devices You are not able to distinguish them and 'link' them to the device in /dev directory. So I can not attach an event to devd(8) to the device that appears in /dev directory.

That is wrong. You can add device file to simple-mtpfs parameters list to get ID of corresponding mtp-compatible device and than supply that ID with --device option to mount only mtp-device you need. Just like:
```
% simple-mtpfs --list-devices /dev/ugen0.5
1: SamsungGalaxy models (MTP)
% sudo simple-mtpfs -o allow_other --device 1 <mount_point>
```
I use this sequence of commands in my own fork of automount that supports mtpfs: https://github.com/wulf7/automount
I can make pull-request if you are intrested in these changes

**Comment by vermaden** [comment]
The automount(8) supports now MTP.
